<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1040</width>
    <height>852</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Работа над ошибками</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_7">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Просмотр по одному</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QPushButton" name="open_btn">
            <property name="text">
             <string>Открыть файл</string>
            </property>
            <property name="icon">
             <iconset resource="images/files.qrc">
              <normaloff>:/icons/open.svg</normaloff>:/icons/open.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="save_btn">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Сохранить файл</string>
            </property>
            <property name="icon">
             <iconset resource="images/files.qrc">
              <normaloff>:/icons/save.svg</normaloff>:/icons/save.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QRadioButton" name="accept">
              <property name="text">
               <string>Ответ принят</string>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">buttonGroup</string>
              </attribute>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="irrelevant">
              <property name="text">
               <string>Не актуальный</string>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">buttonGroup</string>
              </attribute>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLabel" name="score_lb">
                <property name="text">
                 <string>  Оценка</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="score">
                <item>
                 <property name="text">
                  <string>2</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>1</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>0</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>-1</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>-2</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QCheckBox" name="filter_comments_cb">
              <property name="text">
               <string>Показать все</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="comment_lb">
            <property name="font">
             <font>
              <family>Arial</family>
              <pointsize>8</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Комментарий</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>Инстракт</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPlainTextEdit" name="instruct">
                <property name="font">
                 <font>
                  <family>Consolas</family>
                  <pointsize>10</pointsize>
                 </font>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_2">
                <item>
                 <widget class="QLabel" name="label_2">
                  <property name="text">
                   <string>Программа</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QTableView" name="code_table">
                  <property name="font">
                   <font>
                    <family>Consolas</family>
                    <pointsize>10</pointsize>
                   </font>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Ответ тренера</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="triple_tik_couch">
                <property name="text">
                 <string>```</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="copy_couch_to_answer">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Копировать в новый</string>
                </property>
                <property name="icon">
                 <iconset resource="images/files.qrc">
                  <normaloff>:/icons/move.svg</normaloff>:/icons/move.svg</iconset>
                </property>
                <property name="iconSize">
                 <size>
                  <width>24</width>
                  <height>24</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPlainTextEdit" name="couch_answer">
              <property name="font">
               <font>
                <family>Consolas</family>
                <pointsize>10</pointsize>
               </font>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Мой ответ</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_4">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="triple_tik_my">
                <property name="text">
                 <string>```</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="copy_my_to_answer">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Копировать в новый</string>
                </property>
                <property name="icon">
                 <iconset resource="images/files.qrc">
                  <normaloff>:/icons/move.svg</normaloff>:/icons/move.svg</iconset>
                </property>
                <property name="iconSize">
                 <size>
                  <width>24</width>
                  <height>24</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPlainTextEdit" name="my_answer">
              <property name="font">
               <font>
                <family>Consolas</family>
                <pointsize>10</pointsize>
               </font>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Новый ответ</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPlainTextEdit" name="new_answer">
              <property name="font">
               <font>
                <family>Consolas</family>
                <pointsize>10</pointsize>
               </font>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QLabel" name="counter_lb">
                <property name="font">
                 <font>
                  <pointsize>10</pointsize>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>_</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="previous">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Предыдущий</string>
                </property>
                <property name="icon">
                 <iconset resource="images/files.qrc">
                  <normaloff>:/icons/back.svg</normaloff>:/icons/back.svg</iconset>
                </property>
                <property name="iconSize">
                 <size>
                  <width>24</width>
                  <height>24</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="next">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="layoutDirection">
                 <enum>Qt::RightToLeft</enum>
                </property>
                <property name="text">
                 <string>Следующий</string>
                </property>
                <property name="icon">
                 <iconset resource="images/files.qrc">
                  <normaloff>:/icons/forward.svg</normaloff>:/icons/forward.svg</iconset>
                </property>
                <property name="iconSize">
                 <size>
                  <width>24</width>
                  <height>24</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Просмотр таблицей</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QTableView" name="full_table_tv">
          <property name="sizeAdjustPolicy">
           <enum>QAbstractScrollArea::AdjustToContents</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="images/files.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
